index.html

```diff
@@ -362,7 +362,41 @@ <h1 class="aboutme">About me</h1>
             </main>
       
 
-   
+   <!-- contact form -->
+<section class="contact_form">
+  <div class="contact_title">
+    <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+
+  
+  </div>
+  <form action="https://formspree.io/f/mqkoykza" method="post">
+    <ul>
+    <li><label for="name">
+      
+     <input type="text" name="name" id="name" placeholder="Full name" maxlength="30" required>
+    </label></li>
+    <li><label for="email">
+      
+      <input type="email" name="email" id="email" placeholder="Email address" required>
+    </label></li>
+    <textarea name="" placeholder="Write me something...." maxlength="500"></textarea>
+    </ul>
```

Comment:
- [ ] Please make sure that all form fields should have `required` attribute including `textarea` as it is the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) of the project that a user should not be able to submit the form if one or more of the fields is empty

> **Project Requirements**
>
>  Make sure that:
> - A user cannot submit the form if one or more of the fields is empty.